test/utils.js

```diff
@@ -783,3 +783,56 @@ export async function setupMetricsTransactions () {
 
   await MetricModel.insertMany(metrics)
 }
+
+import { connectionDefault } from '../src/config'
+
+let bucket
+const getGridFSBucket = () => {
+  if (!bucket) {
+    bucket = new mongodb.GridFSBucket(connectionDefault.client.db())
+    return bucket
```

Comment:
@BMartinos Same as before - this return is redundant

Reply:
Done in 
f6cb580
ad6cbc9